<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>listmenu</class>
 <widget class="QWidget" name="listmenu">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>686</width>
    <height>809</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#listmenu{
border-image: url(:/newPrefix/list.jpg);
}</string>
  </property>
  <widget class="QPushButton" name="pushButton_4">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>70</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>8514oem</family>
     <pointsize>17</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">.QPushButton
{
background: rgb(170,170, 255);

font: 17pt &quot;8514oem&quot;;
border-radius: 10px;
}

.QPushButton:hover
{
 	background-color: rgb(100, 150, 255);
}
</string>
   </property>
   <property name="text">
    <string>BACK</string>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_5">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>70</y>
     <width>131</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>8514oem</family>
     <pointsize>18</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">.QPushButton
{
background: rgb(170,170, 255);

font: 18pt &quot;8514oem&quot;;
border-radius: 10px;
}

.QPushButton:hover
{
 	background-color: rgb(100, 150, 255);
}
</string>
   </property>
   <property name="text">
    <string>LOGOUT</string>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QTableWidget" name="tableWidget">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>150</y>
     <width>311</width>
     <height>411</height>
    </rect>
   </property>
   <column>
    <property name="text">
     <string>name</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>cost</string>
    </property>
   </column>
  </widget>
  <widget class="QPushButton" name="pushButton_9">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>590</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">.QPushButton
{
background: rgb(170,170, 255);

font: 15pt &quot;8514oem&quot;;
border-radius: 10px;
}

.QPushButton:hover
{
 	background-color: rgb(100, 150, 255);
}
</string>
   </property>
   <property name="text">
    <string>DISPLAY</string>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_6">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>590</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">.QPushButton
{
background: rgb(170,170, 255);

font: 15pt &quot;8514oem&quot;;
border-radius: 10px;
}

.QPushButton:hover
{
 	background-color: rgb(100, 150, 255);
}
</string>
   </property>
   <property name="text">
    <string>ADD</string>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_7">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>590</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">.QPushButton
{
background: rgb(170,170, 255);

font: 15pt &quot;8514oem&quot;;
border-radius: 10px;
}

.QPushButton:hover
{
 	background-color: rgb(100, 150, 255);
}
</string>
   </property>
   <property name="text">
    <string>DELETE</string>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>660</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_2">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>660</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="resim/Yeni Metin Belgesi.qrc"/>
 </resources>
 <connections/>
</ui>
